Src/Newtonsoft.Json/Converters/JavaScriptDateTimeConverter.cs

```diff
@@ -111,7 +111,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 throw JsonSerializationException.Create(reader, "Unexpected token parsing date. Expected EndConstructor, got {0}.".FormatWith(CultureInfo.InvariantCulture, reader.TokenType));
             }
 
-#if !NET20
+#if HAVE_NULLABLE
```

Comment:
Every build should have nullable. I'm confused why this even has an `#if` around it.